src/index.html

```diff
@@ -19,9 +19,666 @@
       rel="stylesheet"
       href="styles/index.scss"
     />
+    <link
+      rel="stylesheet"
+      href="styles/HeaderStyle/_header.scss"
+    />
+    <link
+      rel="stylesheet"
+      href="styles/CardStyle/main.scss"
+    />
+    <link
+      rel="stylesheet"
+      href="styles/CatalogStyle/_catalog.scss"
+    />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list-item">
+            <a
+              href="#apple"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="#samsung"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="#smartphones"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="#laptopscomputers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="#gadgets"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="#tablets"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="#photo"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="#video"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!-- Sekcja główna -->
+    <main>
+      <!-- pierwsza karta -->
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="img-imac">
+          <img
+            src="./images/imac.jpeg"
+            alt="imac"
+          />
+        </div>
```

Comment:
you don't need container for image and it's better to name class as element of card block. remember to change it for every card :)
```suggestion
        <img
          class="card__img"
          src="./images/imac.jpeg"
          alt="imac"
        />
```